src/features/schedule/ScheduleCardDayMenu.tsx

```diff
@@ -16,8 +17,11 @@ const ScheduleCardDayMenu = ({ selectedDay, nextFiveDays, setSelectedDay, daysWi
           }}
           selected={isSameDay(day, selectedDay)}
         >
-          <span>{daysWithEvents.includes(day) ? '\u2022' : ''}</span>
-          <span>{format(day, 'ddd')}</span>
+          <span aria-hidden>{daysWithEvents.includes(day) ? '\u2022' : ''}</span>
+          <span>
+            <span aria-hidden>{format(day, 'ddd')}</span>
+            <VisuallyHidden>{format(day, 'dddd')}</VisuallyHidden>
```

Comment:
Should the date format be consistent here?

Reply:
They are different on purpose. Screen readers will now read: “Monday” while the visual UI date remains Mon/MON.